src/app/start/startdesk.component.ts

```diff
@@ -75,7 +75,7 @@ export class StartDeskComponent implements OnInit {
 
     ngOnInit() {
         this.rmmapi.getFromAddress().subscribe(
-            froms => this.ownAddresses.next(new Set(froms.map(f => f.email))),
+            froms => this.ownAddresses.next(new Set(froms.map(f => f.email.toLowerCase()))),
```

Comment:
I `.toLowerCase`-d the froms in ownAddresses while they're being loaded for this component as well now. Could save some issues with matching/filtering in the future. Thanks @tadzik :)